ocaml/xapi/xapi_vm_migrate.ml

```diff
@@ -1549,8 +1549,8 @@ let migrate_send' ~__context ~vm ~dest ~live:_ ~vdi_map ~vif_map ~vgpu_map
         try
           let _, state = XenopsAPI.VM.stat dbg vm_uuid in
           if Xenops_interface.(state.Vm.power_state = Suspended) then (
-            debug "xenops: %s: shutting down suspended VM" vm_uuid ;
-            Xapi_xenops.shutdown ~__context ~self:vm None
+            debug "xenops: %s: Resuming suspended VM on source host" vm_uuid ;
+            Xapi_xenops.fast_resume ~__context ~self:vm
           )
         with _ -> ()
```

Comment:
It would be useful to log any exceptions here, perhaps at debug level instead of silently eating it up if we want to do anything other than killing the VM here.

Reply:
Yes, that would be good to avoid leaving a VM around in a broken state.